rest-api-spec/src/main/resources/rest-api-spec/test/search/340_doc_values_field.yml

```diff
@@ -71,6 +71,166 @@
           - '{ "index": { "_index": "test-iodvq", "_id": "3" } }'
           - '{ "some_keyword": "5", "byte": 122, "double": 102.0, "float": "802.0", "half_float": "402.0", "integer": 1292, "long": 13458, "short": 152, "unsigned_long": 10223372036854775802, "ip_field": "192.168.0.3", "boolean": false, "date_nanos": "2024-10-29T12:12:12.987654321Z", "date": "2024-10-29T12:12:12.987Z" }'
 
+  - do:
```

Comment:
Please add to all new test cases:

```
  - skip:
      version: " - 2.16.99"
      reason: "the bug was fixed since 2.17.0"

```

Otherwise the test cases will fail in mixed clusters